tests/loggers/test_wandb.py

```diff
@@ -18,10 +18,16 @@
 
 import pytest
 
+# ensure we consider recent wandb versions
+import pytorch_lightning.loggers.wandb as pl_wandb
 from pytorch_lightning import Trainer
 from pytorch_lightning.loggers import WandbLogger
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
 from tests.helpers import BoringModel
+from tests.helpers.utils import no_warning_call
+
+pl_wandb._WANDB_GREATER_EQUAL_0_10_22 = True
+pl_wandb._WANDB_GREATER_EQUAL_0_12_10 = True
```

Comment:
is it possible to mock them instead for the test?

Reply:
yep, monkeypatch.setattr would be a better way to mock the attribute